contrib/window_tshirt_sizes/src/sizes.cpp

```diff
@@ -0,0 +1,139 @@
+
+#include <cassert>
+#include <stdio.h>
+
+class SDL_Point {
+public:
+	int x            = 0;
+	int y            = 0;
+	const char* name = nullptr;
+	constexpr SDL_Point(const int _x, const int _y, const char* _name = nullptr)
+	        : x(_x),
+	          y(_y),
+	          name(_name){};
+};
+
+constexpr SDL_Point eight_by_five_ratio = {8, 5, "widescreen"};
+constexpr SDL_Point four_by_three_ratio = {4, 3, "standard"};
+
+constexpr SDL_Point list_of_aspect_ratios[] = {eight_by_five_ratio,
+                                               four_by_three_ratio};
+
+constexpr int small_percent  = 33;
+constexpr int medium_percent = 50;
+constexpr int large_percent  = 75;
+constexpr int full_percent   = 100;
+
+constexpr int list_of_percent_scalers[] = {small_percent,
+                                           medium_percent,
+                                           large_percent,
+                                           full_percent};
+
+// https://en.wikipedia.org/wiki/Display_resolution
+constexpr SDL_Point list_of_screen_dimensions[] = {
+        {640, 480, "VGA"},
+        {720, 480, "480p NTSC"},
+        {726, 576, "480p PAL"},
+        {800, 600, "SVGA"},
+        {1024, 768, "XGA"},
+        {1280, 720, "720p"},
+        {1280, 800, "WXGA"},
+        {1280, 1024, "Super-eXtended Graphics Array (SXGA)"},
+        {1360, 768, "High Definition (HD)"},
+        {1366, 768, "High Definition (HD)"},
+        {1440, 900, "WXGA+"},
+        {1536, 864, "N/A"},
+        {1600, 900, "High Definition Plus (HD+)"},
+        {1680, 1050, "WSXGA+"},
+        {1600, 1200, "High Definition Plus (HD+)"},
+        {1920, 1080, "Full High Definition (FHD)"},
+        {1920, 1200, "Wide Ultra Extended Graphics Array (WUXGA)"},
+        {2048, 872, "Cinemascope"},
```

Comment:
I don't think film industry specific resolutions should be on this list. Only common monitor resolutions should be included.